install

```diff
@@ -162,8 +162,8 @@ group_chmods = %w( bin etc Frameworks include lib sbin share var
                    map { |d| File.join(HOMEBREW_PREFIX, d) }.select { |d| chmod?(d) }
 # zsh refuses to read from these directories if group writable
 zsh_dirs = %w( share/zsh share/zsh/site-functions )
-user_chmods = zsh_dirs.
-                  map { |d| File.join(HOMEBREW_PREFIX, d) }.select { |d| chmod?(d) }
+                  .map { |d| File.join(HOMEBREW_PREFIX, d) }
```

Comment:
@MikeMcQuaid they likely wouldn't exist if it's a fresh new install. But the installer wouldn't repair them in case of re-running it on top of the existing installation.